addons/mrp/models/product.py

```diff
@@ -118,8 +119,11 @@ def _compute_bom_count(self):
         for product in self:
             product.bom_count = self.env['mrp.bom'].search_count(['|', '|', ('byproduct_ids.product_id', '=', product.id), ('product_id', '=', product.id), '&', ('product_id', '=', False), ('product_tmpl_id', '=', product.product_tmpl_id.id)])
 
+    @api.depends_context('company')
     def _compute_is_kits(self):
         domain = ['&', ('type', '=', 'phantom'),
```

Comment:
```suggestion
        domain = ['&', '&', ('type', '=', 'phantom'),
```
Let's make it fully explicit, as it was before the change